make/autoconf/libraries.m4

```diff
@@ -108,12 +108,6 @@ AC_DEFUN([LIB_SETUP_JVM_LIBS],
       BASIC_JVM_LIBS_$1="$BASIC_JVM_LIBS_$1 -latomic"
```

Comment:
... unrelated to your fix, I noted that this use of libatomic should probably check if we're on zero and a 32-bit CPU, not list all known 32-bit CPUs...